### PR TITLE
Feature - useResponsiveScreenSize hook added

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Checklist:

- [x] I have updated the CHANGELOG with a summary of my changes
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/particles/dimensions/theme.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
+import React from 'react';
+
+import { useWindowSize } from '@kibalabs/core-react';
```
Comment (MoiyadKayda) [inline src/particles/dimensions/theme.ts:3]: need to update core-react version

File: src/particles/dimensions/theme.ts [hunk 2]
```diff
@@ -89,3 +95,31 @@ export const getPaddingSize = (size: PaddingSizeProp, theme: IDimensionGuide): s
...
+export const ResponsiveScreenSizeContext = React.createContext<ScreenSize>(ScreenSize.Base);
+
+const getScreenSizeValue = (size: ScreenSize, theme: IDimensionGuide): number => {
+  if (size === ScreenSize.Base) {
+    return 0;
+  }
+  return Number(getScreenSize(size as string as ScreenSize, theme).replace('px', ''));
+};
+
+export function useResponsiveScreenSize(): ScreenSize {
```
Comment (krishan711) [inline src/particles/dimensions/theme.ts:108]: dont put this here.. put it in util/ in its own file.

Comment (MoiyadKayda) [inline src/particles/dimensions/theme.ts:108]: had to change `ResponsiveScreenSize` to `ScreenSize`

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [MINOR] Updated `Box` props to have `title` prop to have a tooltip
 - [MINOR] Updated `Grid` and `EqualGrid` to have `defaultGutter` prop to change the gutter size
 - [MINOR] Updated `LinkBase` props to have `isFullHeight` prop
+- [MINOR] Updated `Dimension` to have `useResponsiveScreenSize` to get screen size
```
Comment (krishan711) [inline CHANGELOG.md:26]: please update this - the new hook should not be "in" dimension

**Review by krishan711 — APPROVED** [review]

**Pull request merged**